docker/tds/docker-compose.yml

```diff
@@ -135,17 +135,46 @@ services:
       - SBAC_JDBC_PASSWORD
       - JAVA_OPTS=${CONTAINER_JAVA_OPTS}
 
+  #TDS_ExamResultsTransmitter
+  exam-results-transmitter:
+    image: fwsbac/tds-exam-results-transmitter
+    ports:
+      - "32849:8080"
+    depends_on:
+      configuration-service:
+        condition: service_healthy
+      assessment:
+        condition: service_started
+      session:
+        condition: service_started
+      exam:
+        condition: service_started
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health.json"]
+      interval: 15s
+      timeout: 10s
+      retries: 10
+    env_file:
+      - tds-common.env
+      - exam-results-transmitter.env
```

Comment:
I've already added an exam-results-transmitter.env file to the dev and qa environments with profile information.